docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Collect a distributed trace - .NET
+description: Tutorials to collect distributed traces in .NET applications using OpenTelemetry, Application Insights, or ActivityListener
+ms.topic: tutorial
+ms.date: 03/14/2021
+---
+
+# Collect a distributed trace
+
+**This article applies to: ✔️** .NET Core 2.1 and later versions **and** .NET Framework 4.5 and later versions
+
+Instrumented code can create <xref:System.Diagnostics.Activity> objects as part of a distributed trace, but the information
+in these objects needs to be collected into a centralized persistant store so that the entire trace can be
+reviewed later. In this tutorial you will collect the distributed trace telemetry in different ways so that it is
+available to diagnose application issues when needed. See
+[the instrumentation tutorial](distributed-tracing-instrumentation-walkthroughs.md) if you need to add new instrumentation.
+
+## Collect traces using OpenTelemetry
+
+### Prerequisites
+
+- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet) or a later version
```

Comment:
nit: to avoid the misconception that "Collect traces using OpenTelemetry" *requires* .NET Core 3.1, I'd suggest to keep this .NET Core 2.1